lib/bolt/config.rb

```diff
@@ -168,6 +182,7 @@ def initialize(boltdir, config_data, overrides = {})
         'log'                 => { 'console' => {} },
         'plugin_hooks'        => {},
         'plugins'             => {},
+        'project_name'        => false,
```

Comment:
I don't think this is strictly necessary, since `#project_name` will return `nil` if the setting was never set.